CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## Next release
 
+...
+
+## Mypy 1.15 (unreleased)
```

Comment:
Please use Title Case for section headings (also "Next Release" above), for consistency.